### PR TITLE
Updated metadata as per new Redfish schema 2022.1

### PR DESCRIPTION
**Pull request by chauberahul1993**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

File: svc-api/handle/handlers.go [hunk 1]
```diff
@@ -287,7 +287,7 @@ func GetMetadata(ctx iris.Context) {
 			models.Reference{URI: "http://redfish.dmtf.org/schemas/v1/Endpoint_v1.xml",
 				TopInclude: []models.Include{
 					models.Include{Namespace: "Endpoint.v1_0_0"},
```
Comment (amar-shalgar) [inline svc-api/handle/handlers.go:289]: replace Endpoint.v1_0_0 to Endpoint

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: svc-api/handle/handlers.go (hunk 1 above)
Comment (chauberahul1993) [reply svc-api/handle/handlers.go]: DONE

**Review by amar-shalgar — APPROVED** [review]

File: svc-fabrics/README.md [hunk 2]
```diff
@@ -627,7 +627,7 @@ curl -i GET \
 ```
 { 
    "@odata.id":"/redfish/v1/Fabrics/8f6d8828-a21a-464f-abf9-ed062fa08cd9/Endpoints/b21f3e57-e46d-4a8e-92c8-8658edd107cb",
-   "@odata.type":"#Endpoint.v1_11_0.Endpoint",
+   "@odata.type":"#Endpoint.v1_7_0.Endpoint",
```
Comment (shivacharanms) [inline svc-fabrics/README.md:630]: why are we going back in version?

Comment (chauberahul1993) [reply svc-fabrics/README.md:630]: Previously was updated wrong, That is correct

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**